common.gypi

```diff
@@ -597,9 +597,6 @@
       # Enable FILE support by default.
```

Comment:
I don't think this file is relevant to anything anymore. We can probably remove it

Reply:
@bridiver can we also remove filename_rules.gypi, release.gypi and docs-translations/?

Reply:
@bridiver there still some variables used: `v8_use_external_startup_data`, `enable_hidpi`.